packages/webapp/pages/register.tsx

```diff
@@ -12,13 +12,27 @@ import {
   ProfileHeading,
 } from '@dailydotdev/shared/src/components/utilities';
 import AnalyticsContext from '@dailydotdev/shared/src/contexts/AnalyticsContext';
+import FeaturesContext from '@dailydotdev/shared/src/contexts/FeaturesContext';
+import {
+  Features,
+  getFeatureValue,
+} from '@dailydotdev/shared/src/lib/featureManagement';
 import MainLayout from '../components/layouts/MainLayout';
 
 export default function Register(): ReactElement {
   const { user, logout } = useContext(AuthContext);
   const router = useRouter();
   const [disableSubmit, setDisableSubmit] = useState<boolean>(true);
   const { trackEvent } = useContext(AnalyticsContext);
+  const { flags } = useContext(FeaturesContext);
+
+  const getSignupModalFeatureValue = (featureFlag: Features) => {
+    return getFeatureValue(featureFlag, flags);
+  };
+
+  const isFeatureEnabled = (featureFlag: Features) => {
+    return parseInt(getSignupModalFeatureValue(featureFlag), 10);
+  };
```

Comment:
Oh sorry my bad, I see we don't have this setup in the `featureManagement` here.
This should be sitting in there as soon as we merge our current branch.

For now you could add the following code to the `featureManagement` file so we have it aligned.

```js

export const isFeaturedEnabled = (key: Features, flags: IFlags): boolean =>
  flags[key?.id]?.enabled;
```

See the following PR for the new file:
https://github.com/dailydotdev/apps/blob/DD-354-my-feed/packages/shared/src/lib/featureManagement.ts